spec/acceptance_tests/capybara_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'spec_helper'
+
+describe GovukAbTesting::AcceptanceTests::Capybara do
+  let(:driver) { double(options: {}, header: true) }
+  let(:page) {
+    double('Capybara::Session', driver: driver)
+  }
+  let(:scope) { double(page: page) }
+  let(:capybara_acceptance_test) { described_class.new(scope) }
```

Comment:
would it make sense to move `driver`, `page` and `scope` into the `capybara_acceptance_test` block? That makes it clear that they won't be used in the tests themselves.